lib/ar2dto/dto.rb

```diff
@@ -4,13 +4,23 @@ module AR2DTO
   class DTO
     class << self
       def [](original_model)
-        Class.new(self) do |_klass|
+        Class.new(self) do |klass|
+          klass.const_set(:ORIGINAL_MODEL, original_model)
           attr_accessor(*original_model.attribute_names)
         end
       end
     end
 
-    include ::ActiveModel::Model
+    include ::ActiveModel::AttributeAssignment
+    include ::ActiveModel::Conversion
+    extend ::ActiveModel::Naming
+    extend ::ActiveModel::Translation
```

Comment:
Is there a possibility that we declare a module for all the `ActiveModel-specific` stuff?

I feel all this has a clear purpose, make the object behave like an ActiveModel, so if we could wrap it into some module then it could be clear where to find that logic. Moreover we could avoid overriding the rubocop rule, and may (not sure though) be easier to include the feature of deciding whether or not the object behaves like an ActiveModel or not.

What do you think?

Reply:
I completely agree

Reply:
There I tried, but there is 1 problem. the `initialize` behavior will always use `ActiveModel::AttributeAssignment` right? https://github.com/santib/ar2dto/pull/10/files#diff-770268259995cbd7d47488c94bca935333576a0d8d74cc4a637f504563d30fc4R14

Reply:
done